scripts/buildLib.mjs

```diff
@@ -45,11 +45,11 @@ if (watch) {
       libInDir,
       "--out-dir",
       libOutDir,
-      "--watch",
       "--skip-initial-build",
       "--ignore",
       alloyInFile,
-      "--presets=@babel/preset-env",
```

Comment:
Using `--presets=@babel/preset-env` in the CLI command is ignoring any browser config from the Babel config file.